website/pages/resource.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="page-container page-resource">
     <h2>{{ langConfig[1] }}</h2>
-    <p>{{ langConfig[placeholder2] }}</p>
+    <p>{{ langConfig['placeholder2'] }}</p>
```

Comment:
use `langConfig.placeholder2` instead.

Reply:
Shouldn't here be like `2`? instead of `placeholder2`? You can see that here are incremental numbers like `1, 3, 4, 5` I think this should be `2`.

Reply:
No, `resources`'s data structure is in blow:

```
"resource": {
  "1": "Resource",
  "2": "Under construction.",
  "3": "Axure Components",
  "4": "By importing Element UI in Axure, you can easily apply all the components we provide during interaction design.",
  "5": "Download",
  "6": "Sketch Template",
  "7": "Apply components from Element template, so you can improve design efficiency while keeping a unified visual style.",
  "8": "Interaction Design Documentation",
  "9": "Take a closer look at the interaction design documentation. Learn more details of each component from a microcosmic perspective.",
  "paraHeight": "1.6",
  "placeholder1": "Under construction",
  "placeholder2": "More resources are being developed"
}
```

I will change it to `langConfig.placeholder2`.